README.md

```diff
@@ -1,2 +1,73 @@
 # playground-for-go
 Simple examples that help you get started with Appwrite + Go (=❤️)
+
+**Work in progress**
+
+## Get Started
+
+This playground doesn't include any Go best practices but rather it's intended to show some of the most simple examples and use cases of using the Appwrite API in your Go application.
+
+## Requirements
+* A system with Go or Docker installed.
+* An Appwrite instance.
+* An Appwrite project created in the console.
+* An Appwrite API key created in the console.
+
+### Installation
+
+1. Clone this repository.
+2. `cd` into the repository.
+3. Open the `playground.go` file found in the root of the cloned repository.
+4. Copy Project ID, endpoint and API key from Appwrite console into `playground.go`
+5. Run the playground:
+    - Go:
+        - Install dependencies `go get`
+        - Execute the command `go run .`
+    - Docker:
+        - Execute the command `docker compose up`
+6. You will see the JSON response in the console.
+
+### API's Covered
```

Comment:
These are being nested under Requirements which doesn't seem right. Let's just use level 2 headings for both of these.